loleaflet/src/control/Control.MobileWizardBuilder.js

```diff
@@ -19,6 +19,7 @@ L.Control.MobileWizardBuilder = L.Control.JSDialogBuilder.extend({
 		this._controlHandlers['listbox'] = this._listboxControl;
 		this._controlHandlers['checkbox'] = this._checkboxControl;
 		this._controlHandlers['basespinfield'] = this.baseSpinField;
+		this._controlHandlers['PreventNonNumericalInput'] = this._spinFieldPreventNonNumericalInput;
```

Comment:
I think we shouldn't add it here as _controlHandlers should contain only functions building widgets with specific arguments (parent, data, builder)